Query/Dml.php

```diff
@@ -47,6 +47,29 @@
 interface Dml
 {
     /**
+     * Set enclose symbols.
+     *
+     * @param  string  $openSymbol     Open symbol.
+     * @param  string  $closeSymbol    Close symbol.
+     * @return  static
```

Comment:
My IDE support it, but according to [phpdoc.org/types](http://phpdoc.org/docs/latest/references/phpdoc/types.html) you're right,  `static` isn't valid.
Do you prefer `self` or class name ?